auto-configurations/models/chat/memory/spring-ai-autoconfigure-model-chat-memory-jdbc/src/main/java/org/springframework/ai/model/chat/memory/jdbc/autoconfigure/JdbcChatMemoryProperties.java

```diff
@@ -29,16 +29,31 @@ public class JdbcChatMemoryProperties {
 	public static final String CONFIG_PREFIX = "spring.ai.chat.memory.repository.jdbc";
 
 	/**
-	 * Whether to initialize the schema on startup.
+	 * Whether to initialize the schema on startup. Values: embedded, always, never.
+	 * Default is embedded.
 	 */
-	private boolean initializeSchema = true;
+	private String initializeSchema = "embedded";
```

Comment:
Should this be an enum, like https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/sql/init/DatabaseInitializationMode.java?

By making it an enum, it's easier for developers to configure it because they'll get the autocompletion from the IDE about the valid values.

Reply:
Yes!